File: Java/opentraceability/src/main/java/opentraceability/models/events/PersistentDisposition.java
package opentraceability.models.events;

import java.net.URI;
import java.util.ArrayList;

import opentraceability.utility.attributes.OpenTraceabilityArrayAttribute;
import opentraceability.utility.attributes.OpenTraceabilityAttribute;

public class PersistentDisposition {
    @OpenTraceabilityAttribute(ns="", name="unset", sequenceOrder = 1)
    @OpenTraceabilityArrayAttribute(itemType = URI.class)
    public ArrayList<URI> unset = null;

    @OpenTraceabilityAttribute(ns="", name="set", sequenceOrder = 2)
    @OpenTraceabilityArrayAttribute(itemType = URI.class)
    public ArrayList<URI> set = null;
}

File: Java/OpenTraceability/src/main/opentraceability/models/events/persistentdisposition.kt
package opentraceability.models.events

import opentraceability.utility.attributes.OpenTraceabilityArrayAttribute
import opentraceability.utility.attributes.OpenTraceabilityAttribute
import java.net.URI

class PersistentDisposition {

    @OpenTraceabilityAttribute("","unset", 1)
    @OpenTraceabilityArrayAttribute
    var Unset: ArrayList<URI>? = null

    @OpenTraceabilityAttribute("","set", 2)
    @OpenTraceabilityArrayAttribute
    var Set: ArrayList<URI>? = null
}
